etc/ATSCam/primary_hdu.header

```diff
@@ -52,6 +52,7 @@ ATM3PORT=                      / The port the M3 is directing light into
 TRACKSYS=                      / Tracking coordinate system from ATMCS
 RADESYS =                      / Equatorial coordinate system
 FOCUSZ  =                      / Focus Z position from ATAOS/ATHexapod (mm)
+OBJECT  =                      / Name of the observed object
```

Comment:
Can we put it up the top of this block above RA please?

Reply:
Sure.